rmgpy/chemkin.pyx

```diff
@@ -608,8 +608,8 @@ def readReactionComments(reaction, comments, read = True):
                 family = label,
             )
             
-        elif 'Library reaction:' in line or 'Seed mechanism:' in line:
-            label = str(tokens[-1])
+        elif 'Library reaction:' in line or 'seed mechanism:' in line.lower():
+            label = str(tokens[2])
```

Comment:
It seems like the key change is the index for the tokens. Is the change to `Seed mechanism:` necessary? I would prefer not to mix up the parsing for RMG-Py and RMG-Java generated chemkin files.